docs/snippets/defaults_conventional_model.mdx

```diff
@@ -11,6 +11,6 @@ default_network_embodied_emissions_gco2e_per_kb:
     fixed:  0.00
     mobile: 0.00
   sri:
-    fixed:  0.00443
-    mobile: 0.00797
+    fixed:  0.00000443
+    mobile: 0.00000797
```

Comment:
@LinguoMalkavian it would be great to double check the other SRI defaults if we have time to verify we did the correct unit conversions where necessary.

Reply:
I double checked the ones for network power and we had the correct numbers (`default_usage_kwh_per_gb`)

Reply:
Ok the only other sri  defaults are the consumer device numbers and those are correct, assuming the unit is gCO2  per second as is everywhere else.